### PR TITLE
free images and optimizing product page

### PR DESCRIPTION
**Pull request by petya0111**

### CODE REVIEW

File: client/src/components/product-page/ProductPage.js [hunk 1]
```diff
@@ -156,32 +119,32 @@ const ProductPage = () => {
...
                                         rowSpacing={2}>
-                                        {msizes.map((element,index) => {
-                                            let obj = Object.keys(element);
-                                            let size = obj[0];
-                                            let quantity = element[size];
-                                            if (quantity > 5) {
+                                        {product.sizes.map((element,index) => {
+                                            // let obj = Object.keys(element);
+                                            // let size = obj[0];
+                                            // let quantity = element[size];
```
Comment (IvayloSlavchev) [inline client/src/components/product-page/ProductPage.js]: Please remove the comments

Comment (petya0111) [reply client/src/components/product-page/ProductPage.js]: Done

**Review by IvayloSlavchev — APPROVED** [review]
Great job, @petya0111!

**Review by Arabadzhiew — APPROVED** [review]
LGTM

**Pull request merged**